### ISSUE TITLE
make presentations screen-reader friendly

### ISSUE THREAD

**Issue by bmcmahen** [jmpressjs/jmpress.js#79]
Currently embedded jmpress presentations don't appear to be screenreader friendly. I'm looking at adding this functionality myself, but it might be worth implementing into the core functionality of jmpress itself. To make it screenreader friendly, the 'step' should have a tab-index of -1 (allowing for javascript to add focus to the field). When a step is selected (through the keyboard perhaps) focus would be given to that step. For steps that are not currently visible, aria-hidden = true could be used.
